Patch-Source: https://humungus.tedunangst.com/r/azorius/v/cfe9c246e9e5
--
changeset: 537:cfe9c246e9e5
user: Ted Unangst <tedu@tedunangst.com>
files: upgradedb.go
description:
 fix format string

diff -r f3e5d259c544 -r cfe9c246e9e5 upgradedb.go
--- a/upgradedb.go	2024-05-06 00:44:17 -0400 EDT
+++ b/upgradedb.go	2024-05-08 01:27:44 -0400 EDT
@@ -116,7 +116,7 @@ func upgradedb() {
 			var lnk string
 			err := rows.Scan(&postid, &lnk)
 			if err != nil {
-				elog.Fatalf("can't scan link: %s")
+				elog.Fatalf("can't scan link: %s", err)
 			}
 			m := re_linkhost.FindStringSubmatch(lnk)
 			if len(m) > 1 {
